docs/usage/llms/local-llms.mdx

```diff
@@ -84,65 +94,88 @@ INFO:     Application startup complete.
 INFO:     Uvicorn running on http://0.0.0.0:3000 (Press CTRL+C to quit)
 ```
 
+3. Visit `http://localhost:3000` in your browser.
+
+### 6. Configure OpenHands to use the LLM server
+
+Once you open OpenHands in your browser, you'll need to configure it to use the local LLM server you just started.
+
+When started for the first time, OpenHands will prompt you to set up the LLM provider.
+
+1. Click "see advanced settings" to open the LLM Settings page.
+
+![image](./screenshots/07_openhands_open_advanced_settings.png)
+
+2. Enable the "Advanced" switch at the top of the page to show all the available settings.
```

Comment:
These two instructions seem to say the same thing, but maybe is correct if the UI really makes you do it twice?